docs/source/data.rst

```diff
@@ -3,36 +3,145 @@ Data
-* The stations metadata in YAML file format.
-* The stations issue in YAML file format.
-* Some python code to download the datasets using the URLs provided in the JSON files.
-* Some python code to curate the metadata archive.
+-  Station Raw Data:
 
+   -  Stores disdrometer measurements for days, weeks, and years.
+   -  This dataset can be very heavy.
+   -  No central storage is provided.
+
+-  Station Metadata and Issues:
+
+   -  Stores metadata on disdrometer measurements or measurement issues.
+   -  This dataset should be light.
+   -  Central storage is provided in the ``disdro-data`` Git repository.
+   -  The metadata folder contains a YAML metadata file called
+      ``metadata.yml``. It has a ``data_url`` key that references the
+      raw data. Only one zip file per station is allowed.
+
+How to download metadatas and issues ?
+--------------------------------------
 
 You can clone the disdrodb-data repository with
 
-.. code-block:: bash
+.. code:: bash
 
    git clone https://github.com/ltelab/disdrodb-data.git
 
+However, if you plan to add new data or metadata to the archive, first
+fork the repository on your GitHub account and then clone the forked
+repository.
+
+How to upload metadatas and issues?
+------------------------------------
+
+Do you want to contribute to the project with your own data? Great! Just
+follow these steps:
+
+1. Fork the ``disdro-data`` Git repository.
+
+2. Create a new branch.
+
+   .. code:: bash
+
+      git checkout -b "reader-<data_source>-<campaign_name>"
+
+3. Add your data source, campaign names, and station name to the current
+   folder structure.
+
+4. Add your metadata YAML file for each station ``station_name.yml``, in
+   a metadata directory in the campaign directory. We recommend you
+   copy-paste an existing metadata YAML file to get the correct
+   structure.
+
+5. (Optional) Add your issues YAML files, for each station
+   ``station_name.yml``, in an issue directory located in the campaign
+   directory. We recommend you copy-paste an existing metadata YAML file
+   to get the correct structure.
+
+6. Test that the integration of your new dataset functions by deleting
+   your data locally and re-fetching it through the process detailed
+   above.
+
+7. Commit your changes and push your branch to GitHub.
+
+8. `Create a pull
+   request <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request>`__,
+   and wait for a maintainer to accept it!
+
+9. If you struggle with this process, don’t hesitate to raise an
+   `issue <https://github.com/ltelab/disdrodb-data/issues/new/choose>`__
+   so we can help!
+
+How to download raw data locally ?
+--------------------------------------
+
+Prerequisite: You have already set up the folder hierarchy as described
```

Comment:
I would rather say. 
1. First clone the disdrodb-data repository.
2. The retrieve the path to disdrodb-data/DISDRODB: this will be the disdrodb_dir root directory !

Here we could also suggest to add the path to the disdrodb config file (if we create this). 
I think would short a lot the command line routines and easy the use
Then